homeworks/07-mars-viewer/src/components/marsViewer/gallery/galleryFavourites.tsx

```diff
@@ -0,0 +1,42 @@
+import {Dispatch} from 'react'
+import {useDispatch, useSelector} from 'react-redux'
+
+import {MarsViewerState} from '../../../types/state'
+import {changeFavouriteStatus} from './galleryUtils'
+
+import Style from './gallery.module.css'
+
+
+export const MarsViewerFavourites = (): JSX.Element => {
+    const dispatch: Dispatch<any> = useDispatch()
+    const favourites: Set<string> = useSelector((state: MarsViewerState) => state.favourites)
+
+    return (
+        <div className={Style.marsViewer__gallery}>
+            {
+                !favourites.size
```

Comment:
if you can change condition from falsy to truthy better do that, it easier to read `favourites.size` than `!favourites.size` especially with larger ones